source/_integrations/modern_forms.markdown

```diff
@@ -0,0 +1,72 @@
+---
+title: Modern Forms
+description: Instructions on how to integrate a Modern Forms Smart Fan with Home Assistant.
+ha_category:
+  - Fan
+  - Light
+  - Switch
+ha_release:  2021.7
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_quality_scale: none
+ha_codeowners:
+  - '@wonderslug'
+ha_domain: modern_forms
+ha_zeroconf: true
+ha_platforms:
+  - fan
+  - light
+  - switch
+---
+
+[Modern Forms](https://modernforms.com/) has a line of smart WiFi connected fans that allow for cloud or local control of the fan and light.  There is support for individual sleep timers for the fan and light that can be set and cleared independently.
+
+{% include integrations/config_flow.md %}
+
+## Fans
+
+The `modern_forms` integration has support for the Modern Forms fans. This includes directional support, and sleep timer services for the fan.
+
+## Lights
+
+The `modern_forms` integration has support for the Modern Forms fans light.  This includes brightness, and sleep timer services for the light.
+
+## Switches
+
+The `modern_forms` integration provides support for the following toggleable attributes of a fan:
+
+- Away mode - to allow the fan simulate someone being home.
+- Adaptive learning - for allow learning for away mode.
+- Reboot - to reboot the control module.
+
+## Services
+
+### Service `modern_forms.clear_fan_sleep_timer`
+
+This service will clear the sleep timer for the Fan if it has been set. It will not turn off the fan when the timer is cleared.
+
+### Service `modern_forms.clear_light_sleep_timer`
+
+This service will clear the sleep timer for the Light if it has been set. It will not turn off the light when the timer is cleared.
```

Comment:
```suggestion
This service will clear the sleep timer for the light if it has been set. It will not turn off the light when the timer is cleared.
```